packages/demos-babylon/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'eval-source-map',
+  resolve: {
+    modules: ['node_modules'],
+  },
+  plugins: [
+    new CopyPlugin({
```

Comment:
what is this copying? the html/css? I wonder if HTMLWebpackPlugin would be better

Reply:
This is copying HTML, CSS, and the host assets. I spent a little time trying to get HTMLWebpackPlugin to work for this but I couldn't figure out how to get it to use multiple HTML input templates and generate multiple HTML output files. Do you know if it has those capabilities? If so, we can create a ticket adopting it as a future enhancement.

Reply:
ah I see. I think we can add multiple instances of the HTMLWebpackPlugin, use different chunks for each, but not sure it's really buying us anything. Could you instead add a comment here that this is a simpler way to do our multiple HTML pages just so it is clear for future readers?